File: desafio-proway-2/BackEnd/Training_API/Models/Person.cs
using System.Collections.Generic;

namespace Training_API.Models
{
    public class Person
    {
        public Person(int id, string name, string surname)
        {
            this.Id = id;
            this.Name = name;
            this.Surname = surname;
        }
        public int Id { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public IEnumerable<TrainingRoomPerson> TrainingRoomPeople { get; set; }
        // public TrainingRoomPerson TrainingRoomPerson { get; set; }
        public IEnumerable<RestRoomPerson> RestRoomPeople { get; set; }
        // public RestRoomPerson RestRoomPerson { get; set; }
    }
}

File: desafio-proway-2/FrontEnd/Training/src/app/models/Person.ts
import { RestRoomPerson } from "./RestRoomPerson";
import { TrainingRoomPerson } from "./TrainingRoomPerson";

export class Person {
  
  constructor () {
    this.id = 0;
    this.name = '';
    this.surname = '';
  }
  
  id: number;
  name: string;
  surname: string;
}
